code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -106,6 +106,19 @@
 	..()
 	gear_tweaks += gear_tweak_free_color_choice
 
+/datum/gear/head/flowercrown
+	display_name = "flowercrown selection"
+	description = "A set of flowercrowns, perfect for the queen or even the king."
```

Comment:
you know, you could leave the description empty, since it takes the path's description automatically

however it's shitcode so you can't use it for flowercrowns currently HOWEVER it can work with others.

just a heads up.

Reply:
I knew that, I noticed it when putting other stuff in the loadout. Although upon seeing that it didn't showed up the description of the first item, I added one myself.